workspaces/ui-v2/src/components/QueryParametersPanel.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { FC } from 'react';
+import { makeStyles } from '@material-ui/core';
+import { IFieldRenderer, IShapeRenderer, ShapeRenderer } from './ShapeRenderer';
+import { Panel } from './Panel';
+import { FontFamily } from '<src>/styles';
+
+type QueryParameters = Record<string, IFieldRenderer>;
+
+type QueryParametersPanelProps = {
+  parameters: QueryParameters;
+};
+
+// TODO QPB this should move into redux
+export const convertShapeToQueryParameters = (
+  shapes: IShapeRenderer[]
+): QueryParameters => {
+  const queryParameters: QueryParameters = {};
+  if (shapes.length !== 1 || !shapes[0].asObject) {
+    console.error('unexpected format for query parameters');
+    return {};
+  }
+
+  for (const field of shapes[0].asObject.fields) {
+    queryParameters[field.name] = field;
+  }
+
+  return queryParameters;
+};
+
+export const QueryParametersPanel: FC<QueryParametersPanelProps> = ({
+  parameters,
+}) => {
+  const classes = useStyles();
+  return (
+    // TODO QPB add in query parsing strategy here?
+    <Panel header={''}>
```

Comment:
Here we can probably have something informational about:
- How this query parameter was parsed
- How you can structure an example request (or maybe this fits into the showExample feature)